mne/evoked.py

```diff
@@ -158,6 +158,23 @@ def data(self, data):
         """Set the data matrix."""
         self._data = data
 
+    @fill_doc
+    def get_data(self, picks=None):
+        """Get evoked data as 2D array.
+
+        Parameters
+        ----------
+        %(picks_all)s
+
+        Returns
+        -------
+        data : ndarray, shape (n_channels, n_times)
+            A view on evoked data.
```

Comment:
```suggestion
            A view on evoked data.

        Notes
        -----
        .. versionadded:: 0.24
```

Reply:
I squeezed that into the whatsnew commit. thanks